packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -116,6 +116,11 @@ export default class TradeStore extends BaseStore {
     initial_barriers;
     is_initial_barrier_applied = false;
 
+    proposal_req_id = {
+        CALL: null,
+        PUT : null,
```

Comment:
why are we hardcoding put and call in the object?

Reply:
good point, let me remove it